app/ui/browser/actions/external.js

```diff
@@ -101,7 +101,9 @@ export function menuLocationContext(input, dispatch) {
         pageIndex: -1,
         userTyped: loc,
       }));
-      dispatch(navigatePageTo(-1, fixURL(loc)));
+
+      // TODO this doesn't work anyway, we have to pass in webViewController
+      // dispatch(navigatePageTo(-1, fixURL(loc)));
```

Comment:
None of these are tested and probably don't work, going to tackle this next so it's just a list for me heh